nixos/modules/services/web-apps/stirling-pdf.nix

```diff
@@ -100,10 +107,6 @@ in
         ];
         RestrictNamespaces = true;
         RestrictRealtime = true;
-        SystemCallArchitectures = "native";
-        SystemCallFilter = [
-          "~@cpu-emulation @debug @keyring @mount @obsolete @privileged @resources @clock @setuid @chown"
```

Comment:
and this. Why not allow the extra required syscalls?

Reply:
My original thought was to relax the restriction to allow permissions possibly used by Stirling-PDF in future as it keeps adding new features to avoid updating the module frequently.

But from a security's perspective, it makes sense to allow only the currently used functionality. So I'll take your advice.